zone/client.h

```diff
@@ -2255,11 +2277,30 @@ class Client : public Mob
 	void CampAllBots(uint8 class_id = Class::None);
 	void SpawnRaidBotsOnConnect(Raid* raid);
 
+	void LoadDefaultBotSettings();
+	int GetDefaultBotSettings(uint8 setting_type, uint16 bot_setting);
+	int GetBotSetting(uint8 setting_type, uint16 bot_setting);
+	void SetBotSetting(uint8 setting_type, uint16 bot_setting, uint32 setting_value);	
+
 private:
 	bool bot_owner_options[_booCount];
 	bool m_bot_pulling;
 	bool m_bot_precombat;
 
+	uint8 fast_heal_threshold;
+	uint8 heal_threshold;
+	uint8 complete_heal_threshold;
+	uint8 hot_heal_threshold;
+	uint32 fast_heal_delay;
+	uint32 heal_delay;
+	uint32 complete_heal_delay;
+	uint32 hot_heal_delay;
+	uint32 cure_delay;
+	uint8 cure_min_threshold;
+	uint8 cure_threshold;
+	bool illusion_block;
+	uint32 _assistee;
```

Comment:
All of these look unused, can we remove them.

![image](https://github.com/user-attachments/assets/7b060410-9e73-4e86-81ad-6967d97a39ed)


Assistee looks to be bot specific ? Can we name it appropriately